src/components/hotel/index.js

```diff
@@ -30,7 +30,7 @@ class HotelPage extends Component {
     for (var key in amenitiesObject) {
       if (!amenitiesObject.hasOwnProperty(key)) continue;
       if (key === fact) {
-        if (amenitiesObject[key] === typeof boolean || amenitiesObject[key] === 'true' || amenitiesObject[key] === 'false') {
+        if (typeof amenitiesObject[key] === 'boolean' || amenitiesObject[key] === 'true' || amenitiesObject[key] === 'false') {
```

Comment:
Given that all the other conditions assume the value type to be `string`, would it be safer to check `typeof amenitiesObject[key] !== 'string'`?